packages/gatsby-theme-docs/color-presets/index.js

```diff
@@ -16,9 +21,9 @@ module.exports = {
     relativePath: 'merchant-center-user-docs',
     value: require('./merchant-center-user-docs'),
   },
-  platformDeveloperDocs: {
+  tutorialDeveloperDocs: {
```

Comment:
I do not clearly understand why you need to rename this.  Is it just because the files have different names now? 

Background: "tutorial" as a section may not even be sure to stay an own microsite.  it was a voluntary choice to make the naming more abstract.

Reply:
We don't need to rename it, I jut thought it would be clearer that way. But I will keep the name then 👍🏼